plugins/main/public/components/common/wazuh-data-grid/wz-data-grid.tsx

```diff
@@ -11,6 +11,7 @@ import {
   EuiTitle,
   EuiFlyoutBody,
   EuiFlexGroup,
+  EuiLink,
```

Comment:
If not used, it could be deleted

Reply:
Solved in [4216819](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6831/commits/42168192f7c168920b7b6cf111b812f32b894094)